dataproxy/nodejs-mongodb-itx/test/long-running.test.ts

```diff
@@ -36,7 +36,11 @@ describe('long-running', () => {
     })
   })
 
-  testIf(!isDP1)(
+  // While testing we figure out that MongoDB is using DP1
+  // So we made the test a `test.failing` test
+  // TODO: Remove this once we have DP2/accelerate implementation for MongoDB
```

Comment:
We do! It's just not being properly tested, but https://github.com/prisma/ecosystem-tests/pull/3939 does that (by adding the necessary env vars and changing the code to use them)

Reply:
```suggestion
  // TODO: Remove this when https://github.com/prisma/ecosystem-tests/pull/3939 adds testing for DP2/Accelerate
```

Reply:
Oh right ...!

Reply:
Let's clean that up because it's very confusing